Shell/Preprocess.cpp

```diff
@@ -182,11 +204,6 @@ void Preprocess::preprocess(Problem& prb)
     LambdaElimination::addProxyAxioms(prb);
   }
 
-  if (prb.hasInterpretedOperations() || env.signature->hasTermAlgebras()){
-    // Some axioms needed to be normalized, so we call InterpretedNormalizer twice
-    InterpretedNormalizer().apply(prb);
```

Comment:
This was annoying, but I think we were lazy to supply some of the axioms in CNF. 

Not sure what's the situation now. 

Have you considered this?

Reply:
I made sure all axioms are in CNF now :)

Reply:
Perfect!